metrics/grafana/pd.json

```diff
@@ -2240,7 +2240,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(delta(pd_schedule_operators_count{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=\"$instance\", event=\"create\"}[1m])) by (type)",
+              "expr": "sum(rate(pd_schedule_operators_count{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=\"$instance\", event=\"create\"}[1m])) by (type)",
```

Comment:
The `yaxes`  is `opm`, but this formula changes it to `ops`.

Reply:
great catch, changing the units to `ops` on the dashboards and multiplying the expressions for alerts by `60`

Reply:
I think some units should keep `opm`, because some parameters are converted into minutes, for example, we have a parameter of [ `store limit`](https://docs.pingcap.com/tidb/stable/configure-store-limit#view-setting-of-the-current-store). 
I will check it one by one, thanks.

Reply:
Oops, just saw this comment. Let me update the queries to be `opm` then